### PR TITLE
Ensure Greek characters are transliterated

### PR DESCRIPTION
**Pull request by thomasleese**
At the moment there's an issue where Greek characters aren't been transliterated by ActiveSupport, and instead we end up with an empty string:

```rb
> "Στερεά Ελλάδα".parameterize
=> ""
```

This is an issue because this output is used the generate the slug of documents. After this change we get a more usable output:

```rb
> "Στερεά Ελλάδα".parameterize
=> "sterea-ellada"
```

I got the letters from https://gist.github.com/Achillefs/925267 and I'm not sure about the validity of them.

This has come up as part of the "Protected Food & Drink Names" finder where some titles are composed entirely of Greek letters and aren't able to be saved correctly because the slug ends up being an empty string.

### CODE REVIEW

**Review by ChrisBAshton — APPROVED** [review]
LGTM. Is it worth writing a test that `.parameterize` now does what you want it to? (You've already done it in the commit message, may as well put it in a unit test if possible, for extra confidence).

I hadn't come across Rails transliteration before. Nice one 👍

**Comment by thomasleese** [comment]
> LGTM. Is it worth writing a test that `.parameterize` now does what you want it to? (You've already done it in the commit message, may as well put it in a unit test if possible, for extra confidence).

Yep, good point, I've pushed up a commit with a test.

File: config/locales/en.yml [hunk 1]
```diff
@@ -32,6 +32,108 @@
...
+        φ: f
+        Φ: f
+        χ: ch
+        Χ: ch
+        ψ: ps
+        Ψ: ps
+        ω: w
+        ώ: w
+        Ώ: w
+        Ω: w
```
Comment (richardTowers) [inline config/locales/en.yml:99]: These ones are the only ones that stand out as possibly wrong to me. I think Omega is usually a long "o" sound (like in "toe"), so it should probably transliterated as "o", not "w".

Comment (richardTowers) [reply config/locales/en.yml:99]: From my extensive duolingo greek 😬 

I'm sure we could find a native speaker if we wanted to be sure.

Comment (benjamineskola) [reply config/locales/en.yml:99]: I also wondered about `ου` and `ευ` which if I remember correctly depend on the consonant that follows them — but I suppose it's close enough to be recognisable and it's pretty unambiguous.

Comment (ChrisBAshton) [reply config/locales/en.yml:99]: Good spot! I guess it makes little difference in practice though, as it's only used in base_path creation.

Comment (thomasleese) [reply config/locales/en.yml:99]: Thanks for this, I'll trust your judgement and open a PR to change the letters for omega!

Comment (thomasleese) [reply config/locales/en.yml:99]: https://github.com/alphagov/specialist-publisher/pull/1771

**Pull request merged**